pkg/runner/cluster_k8s.go

```diff
@@ -646,3 +647,21 @@ func (c *ClusterK8sRunner) maxPods() (int, error) {
 
 	return pods, nil
 }
+
+// Terminates all pods for with the label testground.purpose: plan
+// This command will remove all plan pods in the cluster.
+func (c *ClusterK8sRunner) TerminateAll() error {
+	log := logging.S()
+	client := c.pool.Acquire()
+	defer c.pool.Release(client)
+
+	planPods := metav1.ListOptions{
+		LabelSelector: "testground.purpose=plan",
+	}
+	err := client.CoreV1().Pods(c.config.Namespace).DeleteCollection(&metav1.DeleteOptions{}, planPods)
+	if err != nil {
+		log.Errorw("Could not terminate all pods.", err)
```

Comment:
I think `Errorw` expected an odd number of params: the first param is general msg, and the following params are key-value pairs.

Maybe change to:

```
log.Errorw("could not terminate all pods", "err", err)
```

nitpick: please use lower-case, as this is what we use everywhere for Infow, Warnw and Errorw.